GNUmakefile

```diff
@@ -58,7 +55,8 @@ SHELL-NAME := makes go-yaml
 include $(MAKES)/clean.mk
 include $(MAKES)/shell.mk
 
-MAKES-CLEAN += $(dir $(YTS-DIR)) $(GOLANGCI-LINT)
+MAKES-CLEAN := $(CLI-BINARY) $(GOLANGCI-LINT)
```

Comment:
Does not including `$(YTS-DIR)` in `MAKES-CLEAN` change the behavior here?

Reply:
YTS-DIR is in realclean.

distclean calls realclean calls clean.